### PR TITLE
[raftstore-v2] add split log

### PR DESCRIPTION
**Pull request by tonyxuqqi**
### What is changed and how it works?

Issue Number: Ref #15056

What's Changed:

```commit-message
Add logs for split region so that we know the time a split takes end-to-end: from the split proposed to the last splitt region finishes the initialization. 
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- No code

### Release note 

```release-note
None
```

### CODE REVIEW

File: components/raftstore-v2/src/operation/command/admin/split.rs [hunk 1]
```diff
@@ -908,7 +913,14 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
         let mut found = false;
         for (_, ids) in self.split_trace_mut() {
             if ids.remove(&region_id) {
+                let ids_len = ids.len();
                 found = true;
+                info!(
```
Comment (bufferflies) [inline components/raftstore-v2/src/operation/command/admin/split.rs:918]: maybe too many logs

**Review by bufferflies — COMMENTED** [review]
rest lgtm

File: components/raftstore-v2/src/operation/command/admin/split.rs (hunk 1 above)
Comment (tonyxuqqi) [reply components/raftstore-v2/src/operation/command/admin/split.rs:918]: We have other logs in split anyway, this does not increase a lot logs. And in v2, split will not be as frequent as v1. So it's likely OK.

**Review by SpadeA-Tang — APPROVED** [review]

**Review by bufferflies — APPROVED** [review]

**Pull request merged**